sdk/eventgrid/azure-resourcemanager-eventgrid/src/test/java/module-info.java

```diff
@@ -1,15 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-open module com.azure.resourcemanager.eventgrid {
```

Comment:
Do we have live tests running successfully without this module-info.java?

Reply:
Does Resource Manager have proper live tests? If so, I'll run them